metrics/rouge/rouge.py

```diff
@@ -55,10 +55,10 @@
     references: list of reference for each prediction. Each
         reference should be a string with tokens separated by spaces.
 Returns:
-    rouge1: rouge_1 f1,
-    rouge2: rouge_2 f1,
-    rougeL: rouge_l f1,
-    rougeLsum: rouge_l precision
+    rouge1: rouge_1 (precision, recall, f1),
```

Comment:
Do you mind adding this to the Args doc just above?
```
      rouge_types: A list of rouge types to calculate. Valide names: `"rouge{n}"` (e.g. `"rouge1"`, `"rouge2"`) where {n} is the n-gram based scoring, `"rougeL"`: Longest common subsequence based scoring. `"rougeLSum"`: rougeLsum splits text using `"\n"`. See details in https://github.com/huggingface/datasets/issues/617
      use_stemmer: Bool indicating whether Porter stemmer should be used to strip word suffixes.
      use_agregator: Return aggregates if this is set to True
```

Reply:
Yes indeed